<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newStyleDlg</class>
 <widget class="QDialog" name="newStyleDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>396</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>396</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>396</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新建模板</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>样式名称:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="styleNameEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>        背景:</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>文件:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileNameEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="openButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>宽度:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>高度:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="widthSpinBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string> 像素</string>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="heightSpinBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string> 像素</string>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>颜色:</string>
      </property>
     </widget>
     <widget class="QToolButton" name="selectColorButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>47</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>图片</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>纯色</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="previewLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>好</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
